quinn-udp/src/fallback.rs

```diff
@@ -26,12 +26,7 @@ impl UdpSocketState {
         }
     }
 
-    pub fn send(
-        &self,
-        socket: UdpSockRef<'_>,
-        _state: &UdpState,
-        transmits: &[Transmit],
-    ) -> io::Result<usize> {
+    pub fn send(&self, socket: UdpSockRef<'_>, transmits: &[Transmit]) -> io::Result<usize> {
```

Comment:
So we have no platforms in CI where this path is used? Maybe we should either get one or remove this code?

Reply:
Strongly agree that we need CI coverage for this. In fact, I just hacked up udp/src/lib.rs to force use of this manually and found (and fixed) a bunch of other bitrot. Not sure what the best way to achieve that is, but for now let's get this merged.